apps/omg_api/test/api/core_test.exs

```diff
@@ -64,7 +58,7 @@ defmodule OMG.API.CoreTest do
 
   @tag fixtures: [:alice, :bob]
   test "encoded transaction is corrupt", %{alice: alice, bob: bob} do
-    encoded_signed_tx = TestHelper.create_encoded([{1, 2, 3, alice}, {2, 3, 4, bob}], eth(), [{alice, 7}])
+    encoded_signed_tx = TestHelper.create_encoded([alice, bob], [{1, 2, 3}, {2, 3, 4}], [{alice, eth(), 7}])
```

Comment:
I'm against this change in `create_encoded` - one of its purposes was to be able to pass the owner to sign together with the input utxo_position. It also looks like a loooot of changes... why?

It is also super-hard to check the changes in the tests now.

Reply:
I don't like that either, but the thing is that now, we don't care about owner of a particular input utxo. We just care if each input owner signed the transaction. 
For example that:
```
TestHelper.create_encoded([{1, 2, 3, alice}, {2, 3, 4, bob}], [{alice, eth(), 7}])
```
creates a transaction that is valid even if bob is the owner of {1,2,3} and alice is the owner of {2,3,4}.

Reply:
so, this is exactly why the test helper is here - someone who writes a test doesn't care about how signatures are created/verified - alice spends - she signs, bob spends - he signs. 

I still think it is better to not have this huge change in the test because of that. All the changes should be hidden within the test helper. You can for example `dedup` the signatures there, or you can support providing inputs without the `, alice}`, as a backwards-compatible extension of the helper.